IdentityServer/Quickstart/Account/ExternalUserInfo.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace IdentityServer.Quickstart.Account
+{
+    /// <summary>
+    /// The viewmodel for /connect/userinfo
+    /// </summary>
+    public class ExternalUserInfo
+    {
+        /// <summary>
+        /// Gets or sets the sub.
+        /// </summary>
+        /// <value>
+        /// The sub.
+        /// </value>
+        [JsonProperty("sub")]
+        public string sub { get; set; }
+        /// <summary>
+        /// Gets or sets the name.
+        /// </summary>
+        /// <value>
+        /// The name.
+        /// </value>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+        /// <summary>
+        /// Gets or sets the name of the family.
+        /// </summary>
+        /// <value>
+        /// The name of the family.
+        /// </value>
+        [JsonProperty("family_name")]
+        public string FamilyName { get; set; }
+        /// <summary>
+        /// Gets or sets the name of the given.
+        /// </summary>
+        /// <value>
+        /// The name of the given.
+        /// </value>
+        [JsonProperty("given_name")]
+        public string GivenName { get; set; }
+        /// <summary>
+        /// Gets or sets the preferred username.
+        /// </summary>
+        /// <value>
+        /// The preferred username.
+        /// </value>
+        [JsonProperty("preferred_username")]
+        public string PreferredUsername { get; set; }
+        /// <summary>
+        /// Gets or sets the profile.
+        /// </summary>
+        /// <value>
+        /// The profile.
+        /// </value>
+        [JsonProperty("profile")]
+        public string Profile { get; set; }
```

Comment:
What is the profile?
Is this an URL? Or something else?

Reply:
This url does not always adhere to the uri standards and such i parse it to a string.

Reply:
I meant that it is not clear from the property name and the documentation what you mean by "Profile". Is this an URL? If so, please add this in the documentation summary.